### PR TITLE
Update Send Tokens

### PR DESCRIPTION
**Pull request by StoicNorth**
Make copy more concise. @dylanschultzie

### CODE REVIEW

File: docs/cli/secretcli.md [hunk 1]
```diff
@@ -230,46 +230,46 @@ Optionally, you can supply your address within the command as:
 secretcli q account $(secretcli keys show -a <key-alias>)
 ```
 
-::: warning Note
```
Comment (StoicNorth) [inline docs/cli/secretcli.md:233]: Is there a reason why this is lowercase? @dylanschultzie

Comment (StoicNorth) [reply docs/cli/secretcli.md:233]: I answered my own question. Plz ignore the changes to the case for all warning --> Warning changes.

File: docs/cli/secretcli.md [hunk 2]
```diff
@@ -230,12 +230,12 @@ Optionally, you can supply your address within the command as:
...
+::: Warning Note
```
Comment (dylanschultzie) [inline docs/cli/secretcli.md:233]: Missed one!

**Review by dylanschultzie — CHANGES_REQUESTED** [review]

File: docs/cli/secretcli.md (hunk 2 above)
Comment (StoicNorth) [reply docs/cli/secretcli.md]: Nice find

**Review by StoicNorth — COMMENTED** [review]
Update typo in markup

**Review by dylanschultzie — APPROVED** [review]

**Pull request merged**